### PR TITLE
HCMPRE-650, HCMPRE-651 :: Team Performance Charts

### PR DESCRIPTION
**Pull request by Ramkrishna-egov**

### CODE REVIEW

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 1]
```diff
@@ -30858,5 +30858,153 @@
...
+      }
+    ],
+    "chartType": "line",
+    "isCumulative": false,
+    "valueType": "number",
+    "plotLabel": "none",
+    "drillChart": "averageNumberOfHousesSprayedByVillage",
```
Comment (holashchand) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:30995]: Please change drillChart here to none

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 2]
```diff
@@ -30858,5 +30858,153 @@
...
+      }
+    ],
+    "chartType": "line",
+    "isCumulative": false,
+    "valueType": "number",
+    "plotLabel": "none",
+    "drillChart": "averageNumberOfHousesSprayedByAP",
```
Comment (holashchand) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:30921]: Here it should be locality one

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 3]
```diff
@@ -30858,5 +30858,153 @@
...
+    "documentType": "_doc",
+    "action": "division",
+    "aggregationPaths": [
+      "Total houses sprayed",
+      "Total_Spray_Operators"
+    ],
+    "filterKeys": [
+      {
+        "key": "district"
```
Comment (holashchand) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:30930]: here it should be administrativeProvince

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 4]
```diff
@@ -30858,5 +30858,153 @@
...
+        "indexName": "user-sync-index-v1",
+        "aggrQuery": "{\"size\":0,\"aggs\":{\"AGGS\":{\"filter\":{\"bool\":{\"must\":[{\"term\":{\"Data.role.keyword\":\"DISTRIBUTOR\"}},{\"exists\":{\"field\":\"Data.boundaryHierarchy.administrativeProvince.keyword\"}}]}},\"aggs\":{\"AGGS\":{\"terms\":{\"field\":\"Data.boundaryHierarchy.administrativeProvince.keyword\",\"size\":500},\"aggs\":{\"Unique Spray Operators Synced\":{\"scripted_metric\":{\"params\":{\"fieldName\":\"Data.syncedUserId.keyword\"},\"init_script\":\"state.list = []\",\"map_script\":\"if(doc[params.fieldName] != null) \\nstate.list.add(doc[params.fieldName].value);\",\"combine_script\":\"return state.list;\",\"reduce_script\":\"Map uniqueValueMap = new HashMap(); \\n        int count = 0;\\n        for(shardList in states) {\\n          if(shardList != null) { \\n            for(key in shardList) {\\n              if(!uniqueValueMap.containsKey(key)) {\\n                count +=1;\\n                uniqueValueMap.put(key, key);\\n              }\\n            }\\n          }\\n        } \\n        return count;\"}}}}}}}}"
```
Comment (holashchand) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:30914]: where this query is used?

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 5]
```diff
@@ -30858,5 +30858,153 @@
...
+    "chartType": "line",
+    "isCumulative": false,
+    "valueType": "number",
+    "plotLabel": "none",
+    "drillChart": "averageNumberOfHousesSprayedByVillage",
+    "documentType": "_doc",
+    "action": "division",
+    "aggregationPaths": [
+      "Total houses sprayed",
+      "Total_Spray_Operators"
```
Comment (holashchand) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:30963]: No query has this path - for all charts

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 6]
```diff
@@ -30858,5 +30858,153 @@
...
+    ],
+    "filterKeys": [
+      {
+        "key": "district"
```
Comment (holashchand) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:30967]: it should be locality here

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 7]
```diff
@@ -30858,5 +30858,153 @@
...
```
Comment (holashchand) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:31004]: it should be village here

**Review by holashchand — CHANGES_REQUESTED** [review]

**Comment by Ramkrishna-egov** [comment]
Closing this PR and raising another due to conflicts

**Pull request closed without merging**